core/executor/src/error.rs

```diff
@@ -58,6 +58,13 @@ pub enum Error {
 	/// Invalid memory reference.
 	#[display(fmt="Invalid memory reference")]
 	InvalidMemoryReference,
+	/// The runtime must provide a global named `__heap_base` of type i32 for specifying where the
+	/// allocator is allowed to place its data.
+	#[display(fmt="The runtime doesn't provide a global named __heap_base")]
+	HeapBaseNotFoundOrInvalid,
+	/// The runtime WebAssembly module is not allowed to have the `start` function.
+	#[display(fmt="The runtime has the \"start\" function")]
```

Comment:
I thought for the console output it might make sense to use `"start"` instead of start with ``, put a thumbs up on this comment if you feel strongly about this and I will change it!

Reply:
Don't have a strong opinion on this, but have seen this a lot in the rust ecosystem, especially in `rustc`. But often I would like to have a terminal that understands markdown for this :man_shrugging: So, do what you think is the best :)

Reply:
Ticks have an advantage that they don't require escaping and thus are a bit more readable